opentech/apply/projects/models.py

```diff
@@ -6,6 +6,11 @@ class Project(models.Model):
 
     name = models.TextField()
 
+    contact_legal_name = models.TextField(default='')
+    contact_email = models.TextField(default='')
+    contact_address = models.TextField(default='')
```

Comment:
Address field doesn't have a database representation, only the form field. I think this will be fine.
https://github.com/OpenTechFund/opentech.fund/blob/master/addressfield/fields.py

Reply:
That's a Form Field which outputs JSON, should I switch to a JSON column instead?

Reply:
JSON does serialise to and from textfield iirc, so ok for our use case?

I don't think we need the power of a JSON column but it wouldn't harm apart from portability (which we've not really focused on).

Reply:
Ah, I misunderstood your initial comment. I'll leave it as a textfield then!